Monika After Story/game/script-affection.rpy

```diff
@@ -496,6 +496,7 @@ init 15 python in mas_affection:
         # unlocks wardrobe if we have more than one clothes available
         if len(mas_selspr.filter_clothes(True)) > 1:
             store.mas_unlockEventLabel("monika_clothes_select")
+            store.mas_lockEventLabel("monika_outfit")
```

Comment:
add TODO saying that the monika_outfit topic should be modified if more than 1 clothes exist.